src/main/java/com/team4/testingsystem/dto/TestDTO.java

```diff
@@ -0,0 +1,139 @@
+package com.team4.testingsystem.dto;
+
+import com.team4.testingsystem.entities.Test;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+
+public class TestDTO {
+    private String level;
+    private LocalDateTime createdAt;
+    private LocalDateTime finishedAt;
+    private String coach;
+    private List<QuestionDTO> grammarQuestions;
+    private List<QuestionDTO> listeningQuestions;
+    private QuestionDTO essayQuestion;
+    private QuestionDTO speakingQuestion;
+    private String contentFile;
+
+    public TestDTO(String level,
+                   LocalDateTime createdAt,
+                   LocalDateTime finishedAt) {
+        this.level = level;
+        this.createdAt = createdAt;
+        this.finishedAt = finishedAt;
+    }
+
+    public TestDTO(Test test) {
+        level = test.getLevel().getName();
+        createdAt = test.getCreatedAt();
+        finishedAt = test.getFinishedAt();
+        coach = test.getCoach().getName();
```

Comment:
Do we really need just coach's name?
Maybe there should be his ID, or a UserDTO...